### DISCUSSION TITLE
자랑글

### DISCUSSION THREAD

**Discussion by ghdcksgml1** [ghdcksgml1/Algorithm_Study#264] [자유게시판]
<img width="607" alt="스크린샷 2022-07-12 오후 11 31 16" src="https://user-images.githubusercontent.com/79779676/178514202-0e82c128-9d28-4a2b-a84f-178b3600b0ef.png">

네이버 부스트 캠프 붙었음! ㅋㅎㅋㅎ

근데 지금 하고있는 알바랑 프로젝트 때문에 참여 못할것같아서 7월 14일 이전에 포기의사를 밝힐예정;; 너무 아쉽네 ㅜ

**Comment by jusung-c** [comment]
에반데 너무 아쉽다
근데 다음에 해도 또 붙으실듯 👍

**Comment by BanApp** [comment]
축하드립니돠 근데 진짜 붓캠하려면 휴학 or 4-2 아니면 못할듯ㄷㄷ
